chefboost/training/Training.py

```diff
@@ -735,7 +735,7 @@ def reconstructRules(source: str, feature_names, tree_id: int = 0) -> None:
     constructor = "def findDecision(obj): #"
     idx = 0
     for feature in feature_names:
-        constructor = constructor + "obj[" + str(idx) + "]: " + feature
+        constructor = constructor + "obj.iloc[" + str(idx) + "]: " + feature

```

Comment:
rules file is getting obj list as input argument and that line is for explaining the indexes of obj file. we don't need iloc at all.